README.md

```diff
@@ -110,11 +110,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo">Not Available</a>
+## 🚀 Live Demo <a href="http://127.0.0.1:5501/about.html" name="live-demo">GH Comedy Crusade</a>
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- Live Demo Link(http://127.0.0.1:5501/about.html)
```

Comment:
- [ ] As requested by the previous reviewer, kindly use GitHub Pages to deploy your application. You can follow this [article](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) to do that.